src/onepasswordconnectsdk/client.py

```diff
@@ -381,34 +388,34 @@ def sanitize_for_serialization(self, obj):
         return self.serializer.sanitize_for_serialization(obj)
 
 
-def new_client(url: str, token: str, is_async: bool = False) -> Union[AsyncClient, Client]:
+def new_client(url: str, token: str, **kwargs) -> Client:
```

Comment:
This change is not backwards compatible unfortunately, and other existing user scripts will break if we ever released this and those users updated to this new version. To support this new separation we'd have to release a new major version of the SDK, namely 2.0, and I'm not sure it's worth it only to separate the client initialisation function. 

Don't get me wrong, I do agree that this would probably be a better pattern, it's just that I don't believe that this improvement alone is worth releasing a 2.0. Can we maybe revert this client separation and only keep the client config part of this PR?